test/integration/project_filter_test.rb

```diff
@@ -0,0 +1,42 @@
+require "test_helper"
+
+class ProjectFilterTest < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+
+  test "a user can view project filters" do
+    project = create(:project)
+    visit projects_path
+    click_link_or_button("Filter Projects")
+
+    assert_equal projects_path, current_path
+    within("#filters") do
+      assert page.has_content?(project.categories.first.name)
+      assert page.has_content?("$0 - $100")
+      assert page.has_content?("$100 - $500")
+      assert page.has_content?("$500+")
+    end
+  end
+
+  test "a user sees only the category of project selected when
+    coming from root path" do
+
+    #Capybara.default_driver = :webkit
```

Comment:
Incorrect indentation detected (column 4 instead of 3).<br>Missing space after #.